docs/developer/dependencies.rst

```diff
@@ -41,11 +40,21 @@ Run
 Test
 ----
 
-  - all (including optional) dependecies of *Run*
+  - all (including optional) dependencies of *Run*
   - ``mantid-framework`` * (GPL-3.0-or-later) (Mantid interoperability tests are skipped if not present)
   - ``psutil`` (BSD-3-Clause) (used to determine if host has enough memory for certain unit tests)
```

Comment:
I think `psutil` is also no longer needed, see above

Reply:
Removed after checking